Sources/CoreCommands/SwiftTool.swift

```diff
@@ -781,16 +781,16 @@ private func getSharedCacheDirectory(options: GlobalOptions, fileSystem: FileSys
     }
 }
 
-private func getSharedCCSDKsDirectory(
+private func getSharedDestinationsDirectory(
```

Comment:
nit: I would keep the "cross compilation" in the function name here to make it easy to reason about